plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -185,7 +189,48 @@ public Version getVersion() throws IOException {
             if (OK.getStatusCode() != response.getStatus()) {
                 throw getDockerException(response);
             }
-            return parseResponseStreamAsListAndClose(response.getInputStream());
+            return parseResponseStreamAsListAndClose(response.getInputStream(), new TypeToken<List<Image>>() {}.getType());
```

Comment:
If we move creation this TypeToken to the field we should do that for all methods, which used method  parseResponseStreamAsListAndClose (for now we have two such methods, but it can be more). I think method parseResponseStreamAsListAndClose should be ease of use, so we should not create new fields for TypeToken in the docker connector.